packages/client/ui/react-ui/src/components/auth/PasskeyPrompt.tsx

```diff
@@ -19,7 +19,7 @@ const dialogStyles: React.CSSProperties = {
 
 const overlayStyles: React.CSSProperties = {
     background: "rgba(139, 151, 151, 0.2)",
-    backdropFilter: "blur(2px)",
+    backdropFilter: "blur(0.125rem)",
```

Comment:
Small note that rem is not necessary for things like blue, boxShadow, border or dividers as these are always fixed sizes and a fixed visual effect

Reply:
😅

Reply:
alright, reverted some of those changes lol